tests/testthat/test-s3-methods.R

```diff
@@ -49,6 +48,18 @@ test_that("tidy, summary, and print work", {
     1
   )
 
+  capture_output(glance(lmo))
+
+  expect_is(glance(lmo), "data.frame")
```

Comment:
instead of constantly re-calling glance(lmo) every line, create the object once and test that:

```
o <- glance(lmo)

expect_is(o, "data.frame")

```